src/common/resize_bilinear.cpp

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
```

Comment:
Just a random place to comment.

The PR does not seem to include tests for the new functionality. How do I test that it works correctly?

Reply:
The same question applies to performance testing.